gsl_demarches_simplifiees/tests/ds_client/test_ds_client.py

```diff
@@ -32,3 +34,75 @@ def test_launch_graphql_query_token_error_logs_and_raises(caplog):
             excinfo.value
         )
         assert "DS forbidden access : token problem ?" in caplog.text
+
+
+def test_get_demarche_dossiers_with_updated_since_calls_graphql_with_iso_format():
+    """Test that get_demarche_dossiers converts datetime to ISO format for GraphQL query."""
+    client = DsClient()
+    demarche_number = 123
+    updated_since = datetime(2024, 1, 15, 14, 30, 45, tzinfo=timezone.utc)
```

Comment:
J'aime bien utiliser `django.utils.timezone` pour avoir des `datetime` qui ne lèvent pas des warnings à tout bout de champ. (Because la flemme d'expliciter tzinfo partout)